repos/system_upgrade/el7toel8/actors/checkrootsymlinks/actor.py

```diff
@@ -0,0 +1,39 @@
+from leapp import reporting
+from leapp.actors import Actor
+from leapp.libraries.stdlib import run
+from leapp.models import Report
+from leapp.tags import ChecksPhaseTag
+
+
+class CheckRootSymlinks(Actor):
+    """
+    Check if the symlinks /bin and /lib are relative, not absolute.
+
+    After reboot, dracut fails if the links are absolute.
+    """
+
+    name = 'check_root_symlinks'
+    consumes = ()
+    produces = (Report,)
+    tags = (ChecksPhaseTag,)
+
+    def process(self):
+        paths = {
+            '/bin': 'usr/bin',
+            '/lib': 'usr/lib'
+            # '/lib64': 'usr/lib64'
+            # '/sbin': 'usr/sbin'
+        }
+        absolute_paths = [path for path in paths if run(['readlink', path])['stdout'].strip()[0] == '/']
```

Comment:
Any reason to not use `os.path.isabs` ?
https://docs.python.org/3/library/os.path.html#os.path.isabs

Reply:
I didn't know it existed, thanks a lot!